### DISCUSSION TITLE
[Feature] Immitch as screensaver for android mobile app

### DISCUSSION THREAD

**Discussion by Nordtus** [immich-app/immich#6269] [Feature Request]
### The feature

Please include a screen saver option for the Android app, in other words when you dock your phone, it'll display immich content. This can also be used to turn old tablets and phones into digital photo frames.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by theothernt** [comment]
I added Immich library support to my free, open source screensaver called Aerial Views - give it a try.

Search for the app on GitHub if you want the APK, or in the Google Play Store.
